Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Materials/MRTKStandardUnityUI.mat

```diff
@@ -0,0 +1,78 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!21 &2100000
```

Comment:
This should likely be under StandardAssets/Materials

Reply:
> This should likely be under StandardAssets/Materials

I would defer this to @cre8ivepark